doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -929,3 +933,99 @@ When an invalid `transient` parameter is given:
 ```
 
 [Top](#top)
+
+##<a name="section7"></a>Metrics
+###<a name="section7.1"></a>`GET /v1/admin/metrics`
+Gets metrics for a whole Cygnus agent. Specifically:
+
+* Number of <b>incoming transactions</b> (a transaction involves a request and a response).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1383/commits/1ca87236b3c2b0b44789f1a3d2cf6801be79fb8d